CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing to OSRD
+
+## Where to start ?
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome.
+
+If you are brand new to OSRD or open-source development, we recommend going through the [GitHub issues](https://github.com/DGEXSolutions/osrd/issues) tab to find issues that interest you. There are number of issues listed under `good first issue` where you could start out.
+
+## Contribute code
+
+Commit style are detailed in the [osrd.fr/docs](https://osrd.fr/en/docs/guides/contribute/code/).
+
+### Pull request / code review
+
+- Every PR, good or bad, is an act of generosity.
+  Opening with a positive comment will help the author feel rewarded, and your subsequent remarks may be heard more clearly.
+  You may feel good also.
+- Add a description to PRs to explain what they do and why.
+- Each PR must be linked to at least one github issue attached to the PI board to simplify tracking of the work done in a sprint.
+- Pay attention to all the comments made by reviewers and either answer/challenge them or address them before pushing new commits and asking for another review.
+- Assign one maintainer and one developer using github assignment when opening the PR to get reviews.
+- Help the reviewer by following advice given in [mtlynch article](https://mtlynch.io/code-review-love/).
+- Review code following advice given in [osrd.fr/docs](https://osrd.fr/en/docs/guides/contribute/code-reviews/) and [mtlynch article](https://mtlynch.io/human-code-reviews-1/).
+- Make commits of logical and atomic units.
```

Comment:
Maybe elaborate on what atomic means for OSRD

Reply:
I think the notion of atomic commit is well documented and we should not have our interpretation of it but keep it as standard as possible. there is a lot of blogs about this